torchfix/__main__.py

```diff
@@ -97,6 +98,9 @@ def main() -> None:
                     torch_files.append(file)
                     break
 
+    if not torch_files:
+        print("No Python files with torch imports found.", file=sys.stderr)
```

Comment:
Let's not print anything (as it's not exactly "files with torch imports").
Just don't do anything, like `flake8 .` does.

Reply:
Makes sense. Fixing.